src/components/Card.tsx

```diff
@@ -0,0 +1,29 @@
+import { FaAngleUp, FaComment } from 'react-icons/fa'
+
+function Card() {
```

Comment:
Here I merged Cardmobile & Carddesktop together not to duplicate components when it's not needed.

Reply:
Please add props to get title, body, category ...rest to render `Card` component dynamically.